.github/workflows/automerge_staging.yml

```diff
@@ -0,0 +1,87 @@
+# INSTRUCTIONS: To enable or disable this workflow, please go to 
+# https://github.com/JuliaRegistries/General/actions/workflows/automerge_staging.yml
+# and click on the "···" button on the right-hand side.
+# Then click on "Enable workflow" or "Disable workflow".
+
+name: AutoMerge (staging)
+on:
+  pull_request:
+
+# Make sure that the `GITHUB_TOKEN` only has read-only permissions
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions
+permissions: read-all
+
+jobs:
+  AutoMerge_staging:
+    timeout-minutes: 60
+    if: github.event_name != 'pull_request' || github.repository == github.event.pull_request.head.repo.full_name
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        version:
+          - '1'
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+      - uses: julia-actions/setup-julia@0b9b1d2cd24245f151902702d8e73b3f6b910014 # v1.6.0
+        with:
+          version: ${{ matrix.version }}
+      - run: rm -rf .ci/JuliaManifest*
+      - run: rm -rf .ci/Manifest*
+      - name: Cache artifacts
+        uses: actions/cache@1a9e2138d905efd099035b49d8b7a3888c653ca8 # v2.1.5
+        env:
+          cache-name: cache-artifacts
+        with:
+          path: ~/.julia/artifacts
+          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/.ci/Manifest.toml') }}
```

Comment:
Will this clash with the main CI cache? And we aren’t using this manifest here, right? 

I’m not totally sure how these are supposed to be used

Reply:
Hmmm. Maybe @giordano or @SaschaMann can weigh in.